docs/en/ingest-management/release-notes/release-notes-8.8.asciidoc

```diff
@@ -0,0 +1,226 @@
+:kibana-issue: https://github.com/elastic/kibana/issues/
+:kibana-pull: https://github.com/elastic/kibana/pull/
+:beats-issue: https://github.com/elastic/beats/issues/
+:beats-pull: https://github.com/elastic/beats/pull/
+:agent-libs-pull: https://github.com/elastic/elastic-agent-libs/pull/
+:agent-issue: https://github.com/elastic/elastic-agent/issues/
+:agent-pull: https://github.com/elastic/elastic-agent/pull/
+:fleet-server-issue: https://github.com/elastic/fleet-server/issues/
+:fleet-server-pull: https://github.com/elastic/fleet-server/pull/
+
+[[release-notes]]
+= Release notes
+
+This section summarizes the changes in each release.
+
+* <<release-notes-8.8.0>>
+
+Also see:
+
+* {kibana-ref}/release-notes.html[{kib} release notes]
+* {beats-ref}/release-notes.html[{beats} release notes]
+
+// begin 8.8.0 relnotes
+
+[[release-notes-8.8.0]]
+== {fleet} and {agent} 8.8.0
+
+Review important information about the {fleet} and {agent} 8.8.0 release.
+
+[discrete]
+[[new-features-8.8.0]]
+=== New features
+
+The 8.8.0 release Added the following new and notable features.
+
+{fleet}::
+* Added audit logging for core CRUD operations {kibana-pull}152118[#152118]
+* Added modal to display versions changelog {kibana-pull}152082[#152082]
+
+{fleet-server}::
+* Documented how to run the fleet server locally {fleet-server-pull}2212[#2212] {fleet-server-issue}1423[#1423]
+* {fleet-server} now supports file uploads for a limited subset of integrations {fleet-server-pull}1902[#1902]
+* Extended the {fleet-server} actions schema to support signed actions passing to the agent as a part of the agent tamper protection. {fleet-server-pull}2353[#2353]
+* {fleet-server} can now be run in stand-alone mode without needing to check into {kib} {fleet-server-pull}2359[#2359] {fleet-server-issue}2351[#2351]
+* Added support for gathering secret values from files {fleet-server-pull}2459[#2459]
+* Added action APM metadata to help debug agent actions {fleet-server-pull}2472[#2472]
+
+{agent}::
+//* EXAMPLE Add support for feature flags, starting with one to toggle fully qualified domain name (FQDN) reporting in events generated by {agent} components, via the `host.name` field {agent-pull}2218[#2218] {agent-issue}2185[#2185]
+* Added a new StateWatch operating mode for watching the current state of the running {agent} {agent-pull}2252[#2252]
+* Added {agent} actions signature validation {agent-pull}2348[#2348]
+* Added a specification file for the link:https://www.elastic.co/observability/universal-profiling[Universal Profiling Symbolizer] {agent-pull}2401[#2401]
+* Added a specification file for the link:https://www.elastic.co/observability/universal-profiling[Universal Profiling Collector] {agent-pull}2407[#2407]
+* Added support to specify the {fleet-server} `service_token` through a file specified with `service_token_file` {agent-pull}2424[#2424]
+* Added support to specify custom base paths during {agent} installation {agent-pull}2500[#2500]
```

Comment:
This can be removed, we hid this option but forgot to remove the changelog fragment: https://github.com/elastic/elastic-agent/pull/2592

It isn't ready for release yet.